packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/intel/requests/BreachedRequest.java

```diff
@@ -0,0 +1,74 @@
+package cloud.pangeacyber.pangea.intel.requests;
+
+import cloud.pangeacyber.pangea.BaseRequest;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class BreachedRequest extends BaseRequest {
```

Comment:
For all the new models, check out the Vault v2 models for how to use lombok to alleviate the need for boilerplate like builders and accessors. For example, I think request models like this one would just need:
```suggestion

@Data
@EqualsAndHashCode(callSuper = true)
@Jacksonized
@SuperBuilder
public class BreachedRequest extends BaseRequest {
```

Reply:
Would it be okey to mix this implementation with the old one? I'd like to be consistent all across the service